lib/chef/compliance/reporter/cli.rb

```diff
@@ -0,0 +1,67 @@
+class Chef
+  module Compliance
+    module Reporter
+      class Cli
+        def send_report(report)
+          # iterate over each profile and control
+          output = ["\nCompliance report:"]
+          report[:profiles].each do |profile|
+            next if profile[:controls].nil?
+
+            output << " * #{profile[:title]}"
+            profile[:controls].each do |control|
+              next if control[:results].nil?
+
+              output << "      #{control[:title]}"
+              control[:results].each do |result|
+                output << format_result(result)
+              end
+            end
+          end
+          output << "\n"
+          puts output.join("\n")
+        end
+
+        private
+
+        def format_result(result)
+          output = []
+          found = false
+          if result[:status] == "failed"
+            if result[:code_desc]
+              found = true
+              output << "         \033[31m\xE2\x9D\x8C #{result[:code_desc]}\033[0m"
```

Comment:
those codes should be using the tty gem, not just doing in-line escape sequences in strings

Reply:
I did look into the tty-gem, while I agree that there should be a better way of doing this(like checking if TTY::Color.support?) I did not want to introduce a gem dependency. I can remove the special characters, and colours, but they do provide some clarity into the success of the result. There are not that many uses of the tty gems.

So I did remove the hardcoded colours and characters and replaced it with pastel that is already being used elsewhere.

Reply:
i'm fairly certain that knife already depends on the tty gem and that is definitely the direction that we'd like to go with output formatting

Reply:
oh nevermind me, i see you've updated to use Pastel and that's what we're actually using in knife for colors...

Reply:
Yes I checked your codebase for what is being used because I know there is coloured output and adjusted accordingly.

Reply:
I am testing this with the latest Chef 16 and in the audit cookbook(the code is pretty much the same, just a different class in the cookbook)